twolevel_dbm.sh

```
@@ -388,6 +389,71 @@ while read -r subject_scans; do
 
   else
     info "Subject ${i} has only a single file, "${scans[@]}", and does not have a subject wise average, it will only have second-level cross sectional DBM"
+
+    for scan in "${scans[@]}"; do
+      info "Generating Overall Jacobian Determinants"
+      if [[ ${_arg_target_space} == "unbiased" ]]; then
+        if [[ ! -s ${_arg_output_dir}/secondlevel/overall-dbm/jacobian/full/$(basename ${scan} | extension_strip).nii.gz ]]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)